### ISSUE TITLE
Namespace the actions

### ISSUE THREAD

**Issue by taco** [bencripps/react-redux-grid#16]
It would be nice if the grid actions were less generic.

Instead of calls like `SET_DATA` it'd be nice if they read something like `SET_GRID_DATA`.

Not sure if this should be a configurable option, or if it would be better with a simple prefix.

``` js
// possible use
import { configureGrid } from 'react-redux-grid';
configureGrid({ actionPrefix: 'RRGRID' });
```

Maybe this would output `SET_RRGRID_DATA` or `RRGRID_SET_DATA`.

It'd be nice to have a global configuration where you don't have to tell each implementation of grid to do this.

@bencripps Let me know your thoughts.

**Comment by taco** [comment]
I'll make a PR for this once we chat about which way to go on this.

**Comment by bencripps** [comment]
This MR should have been closed a looooooong time ago, per #55
